crates/energy/src/battery/systems.rs

```diff
@@ -0,0 +1,23 @@
+use bevy::prelude::*;
```

Comment:
I would merge `systems.rs`, `component.rs` and `mod.rs` to a single `battery.rs`. We can split it once it grows larger.

There are two reasons for this:

* Currently, the combine length of the modules is below 200 lines and the modules are not super complex. We can always split it later once the code size / complexity grows.
* Thorough the codebase, plugins, their systems and related components are kept together. I prefer that we split the plugin to smaller ones to keep the modules manageable (once we reach that point).